src/rabbit_cli.erl

```diff
@@ -223,3 +229,11 @@ rpc_call(Node, Mod, Fun, Args, Timeout) ->
         Time            -> net_kernel:set_net_ticktime(Time, 0),
                            rpc:call(Node, Mod, Fun, Args, Timeout)
     end.
+
+rpc_call(Node, Mod, Fun, Args, Ref, Pid, Timeout) ->
+    case rpc:call(Node, net_kernel, get_net_ticktime, [], Timeout) of
+        {badrpc, _} = E -> E;
+        Time            -> net_kernel:set_net_ticktime(Time, 0),
```

Comment:
@videlalvaro here the `net_ticktime` of the temporary node started when `rabbitmqctl` is executed is being set to match that of the remote node. This is the same logic of the current implementation.